<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Title Machine</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="Phonon::VideoPlayer" name="video_player"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_play">
             <property name="text">
              <string>Play</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_pause">
             <property name="text">
              <string>Pause</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_stop">
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTextBrowser" name="textbrowser_log">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="v_layout_2">
        <item>
         <layout class="QHBoxLayout" name="h_layout_1">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_add_caption">
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_remove_caption">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table_captions">
          <column>
           <property name="text">
            <string>Text</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Start time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Duration</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_process">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>PROCESS</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::VideoPlayer</class>
   <extends>QWidget</extends>
   <header location="global">phonon/videoplayer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>button_play</sender>
   <signal>clicked()</signal>
   <receiver>video_player</receiver>
   <slot>play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>68</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_pause</sender>
   <signal>clicked()</signal>
   <receiver>video_player</receiver>
   <slot>pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_stop</sender>
   <signal>clicked()</signal>
   <receiver>video_player</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
